postgres/adhoc_db.SQL

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE candy(
+    candy_id INT PRIMARY,
+    name VARCHAR(20),
+);
+
+INSERT INTO candy(candy_id, name)
+Values (candy_id, name);
```

Comment:
This should be the actual command.  Can you create a SQL statement that inserts them all in once go.